flag/guest/cluster/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,65 @@
+package kubernetes
+
+// Kubernetes is a data structure to hold guest cluster Kubernetes specific
+// configuration flags.
+type Kubernetes struct {
```

Comment:
I am a little confused, why do you differ from the other flag structure here and have everything in one package?

Reply:
I guess I tried to be clever / lazy at the same time here. I figured it would be nice for code reader to find the final leaves in the same file when there are many of them. The types also felt sound with `kubernetes` package prefix. I'll divide them to sub-packages so they are consistent.